src/main/java/de/tum/in/www1/artemis/web/rest/ModelingAssessmentResource.java

```diff
@@ -27,54 +22,54 @@
 @RestController
 @RequestMapping("/api")
 public class ModelingAssessmentResource extends AssessmentResource {
-    private final Logger log = LoggerFactory.getLogger(ModelingAssessmentResource.class);
+    private final Logger log = LoggerFactory.getLogger(ModelingAssessmentResource.class); //TODO MJ add logging or remove unused logger
 
     private static final String ENTITY_NAME = "modelingAssessment";
+    private static final String PUT_ASSESSMENT_409_REASON = "Given assessment conflicts with exsisting assessments in the database. Assessment has been stored but is not used for automatic assessment by compass";
+    private static final String PUT_ASSESSMENT_200_REASON = "Given assessment has been saved but is not used for automatic assessment by Compass";
+    private static final String PUT_SUBMIT_ASSESSMENT_200_REASON = "Given assessment has been saved and used for automatic assessment by Compass";
+    private static final String REQ_404_REASON = "Requested ressource does not exist.";
```

Comment:
please move the more general errors into Constants.java (or add a new file for this) so that others can reuse them

Reply:
I found an already available Class ErrorConstants and placed the generic error messages there.